cat/ptx-proxy.cat

```diff
@@ -78,4 +74,8 @@ irreflexive ((rf | fr); cause) as axiom-Causality
 // Definition of paper: A Formal Analysis of the NVIDIA PTX Memory Consistency Model (NVIDIA 2019)
 // let release-pattern = ([W & REL]; po-loc?; [W]) | ([F & ACQ_REL]; po; [W])
 // let acquire-pattern = ([R]; po-loc?; [R & ACQ]) | ([R]; po; [F & ACQ_REL])
+
+// Definition of Alloy: https://github.com/NVlabs/mixedproxy
+// let release-pattern = ([W & REL]; po-loc?; [W]) | ([F & ACQ_REL]; po; [W])
+// let acquire-pattern = ([R]; po-loc?; [R & ACQ])
 // irreflexive ((rf | co | fr)?; cause) as axiom-Causality
```

Comment:
Instead of having this at the end of the file, move it to where `release-pattern` and `acquire-pattern` are defined. Please also provide an extensive description of the following problems:
- The proxy alloy model uses `rf | fr | co` in the axiom, while both the original alloy model and the documentation use `rf | fr`. We follow the documentation.
- `acquire-pattern`: the proxy alloy model has a comment saying `A Read followed by an AcquireFence (or greater)`, but the model misses this. The original alloy model has this, but the definition of `acquire-pattern` still doesnot macth the documentation since the documentation says the first read should be a strong one. We follow the documentation

Reply:
Accepted